deploy/configure.sh

```diff
@@ -53,11 +53,6 @@ echoerr "Generate patches..."
 
 # patch the proxy service configuration
 cat <<EOF > patch-proxy-configmap.yaml
-# ----------------------------------------------------------------------
```

Comment:
😱 😄

Reply:
😄  these files are all temp and generated on the fly 🤷